scripts/travis/run_tests

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env bash
 
-# Ordinarily we'd use set -e here but that breaks travis_wait, so we use exit
+# Ordinarily we'd use set -e here but that breaks travis_wait, so we use return
 # pattern below.
 
 set -v
 
-blt validate:all --no-interaction || exit 1
-travis_wait blt setup --define drush.alias='${drush.aliases.ci}' --environment=ci --no-interaction --verbose || exit 1
-blt tests:all --define drush.alias='${drush.aliases.ci}' --define tests.run-server=true --environment=ci --no-interaction --verbose || exit 1
+blt validate:all --no-interaction || { set +v && return 1; }
```

Comment:
I think this should avoid the repetition
```bash
{
set -v
blt validate:all --no-interaction
travis_wait blt setup --define drush.alias='${drush.aliases.ci}' --environment=ci --no-interaction --verbose
blt tests:all --define drush.alias='${drush.aliases.ci}' --define tests.run-server=true --environment=ci --no-interaction --verbose
set +v
return 0
} || { set +v && return 1 }

```

Reply:
I think that will always return exit code 0. If any of validate/setup/test fail, the script will just keep running, there's nothing to force it to stop running.